whosonfirst/Dockerfile

```diff
@@ -3,16 +3,19 @@ FROM pelias/baseimage
 
 # downloader apt dependencies
 # note: this is done in one command in order to keep down the size of intermediate containers
-RUN apt-get update && apt-get install -y bzip2 && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y bzip2 && apt-get install -y unzip && rm -rf /var/lib/apt/lists/*
```

Comment:
can say `apt-get install -y bzip2 unzip` instead of `apt-get install -y bzip2 && apt-get install -y unzip`